### ISSUE TITLE
Typescript export Problems

### ISSUE THREAD

**Issue by jrwpatterson** [UXPin/uxpin-merge-tools#208]
Hey Guys great work on the product, I'm a little concerned though with the typescript exports. 

Firstly they have to be default exports which we don't want to use as it makes it much much more difficult for IntelliSense to keep up with them. 

The second issue is the Arrow support, you have just added Arrow support which is great. But you are taking your component name from the file name rather the const name. For us, we use Kebab case so you are opinionating a file system that works for you but maybe not other people. As for example if I have a component in a folder simple-select.tsx it is being imported as `import simple-select from  ...`

The struggles we are facing is that Storybook and our style guides say we should be exporting named arrow components and our file system should be Kabab case. And so far the only way to do this is to hack, 

if we had a file name action-button.tsx
```
export type SomeProps = {}
/**
 * @uxpincomponent
 */
export function ActionButton(props: SomeProps){
 return <button>{props.label}</button>
}

export default ActionButton
``` 

then in the index file 
```
import { SomeProps, ActionButton } from './action-button'

export {
   SomeProps,
   ActionButton
}

```

with these two errors its not a great experience

**Comment by jrwpatterson** [comment]
I have created pr https://github.com/UXPin/uxpin-merge-tools/pull/209 to fix the naming issue.

**Comment by matthewchigira** [comment]
Fixed by #209
